### PR TITLE
Update README for development Instructions

### LINKED ISSUE #281: Write a readme that explains steps to build & test
`npm install` and `npm run test` did not work for me magically. I discovered that because of optional chaining this depends on Node >=14 (#279) and after I upgraded, I managed to build. But still, I'm getting errors when running tests.

In short, it would be very helpful to write a readme that explains what are the steps to build and test this source code.

### PR DESCRIPTION
**Pull request by ChaituVR**
Fixes #281

### CODE REVIEW

**Comment by ChaituVR** [comment]
@ferittuncer that's weird, what is your node version 🤔

**Comment by ferittuncer** [comment]
@ChaituVR I tested with 14, 12, and 10, all of them gave different errors. Most close to 'successful' was with Node 14, and the screenshot above belongs to that run.

**Comment by ChaituVR** [comment]
> @ChaituVR I tested with 14.16.0, 12.21.0, and 10.24.0, all of them gave different errors. Most close to 'successful' was with Node 14, and the screenshot above belongs to that run.

So, I can't reproduce this error in my local, not sure what's happening within your local, but if this problem persists I suggest you create a nodejs file, and  you can test your strategy by calling this function directly
```
 return snapshot.utils.getScoresDirect(
    'yam.eth',
    [example.strategy],
    example.network,
    new JsonRpcProvider(networks[example.network].rpc[0]),
    example.addresses,
    example.snapshot
  );
  ```

**Comment by ferittuncer** [comment]
To prove that the development instructions in this pull are adequate, you need to start with a fresh environment and then try to run Snapshot. Likely, there is something already installed or configured in your current environment that is forgotten in dependency or instructions.  

Meanwhile, I'll try to call that function, thank you.

**Comment by ChaituVR** [comment]
> To prove that the development instructions in this pull are adequate, you need to start with a fresh environment and then try to run Snapshot. Likely, there is something already installed or configured in your current environment that is forgotten in dependency or instructions.
> 
> Meanwhile, I'll try to call that function, thank you.

Hi, so I tried cloning this repo in Codesandbox and Gitpod, then ran all the scripts mentioned in this PR, everything worked fine 
image
I checked what other dependencies exist in this and found out none, maybe you can use these two services to test this

Then I created a new droplet on Digitalocean with `ubuntu 20.04 (LTS) x64`
Installed Node by following https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04
then I updated the node version to 14.0 with nvm


Cloned snapshot.js and ran install build and test commands and everything seems working
image


Maybe something is wrong with your local dependencies? please check if some global npm dependency is effecting,
 
let me know thanks

**Comment by ferittuncer** [comment]
Yeah, I just tried on a virtual machine, indeed it is working. Weird. 

Let's merge this, as we verified the problem is in my environment.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,27 @@ You can create an index.html file and include Snapshot.js with:
...
+```bash
+npm install
+```
+
+#### Build package
+```bash
+npm run build
+```
+
+#### Test Strategy
```
Comment (bonustrack) [inline README.md:28]:
```suggestion
#### Test strategy
```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,27 @@ You can create an index.html file and include Snapshot.js with:
 ```html
 <script src="https://cdn.jsdelivr.net/npm/@snapshot-labs/snapshot.js"></script>
 ```
+### Development
+#### Install dependencies
```
Comment (bonustrack) [inline README.md:18]:
```suggestion

#### Install dependencies
```

**Review by bonustrack — APPROVED** [review]

**Pull request merged**